samples/examples/income_classifier/infer-k8s.md

```diff
@@ -0,0 +1,298 @@
+    x_outlier = np.load(f)
+```
+
+```python
+reqJson = json.loads('{"inputs":[{"name":"input_1","data":[],"datatype":"FP32","shape":[]}]}')
+url = "http://"+MESH_IP+"/v2/models/model/infer"
+url
+```
+
+```
+'http://172.19.255.1/v2/models/model/infer'
+
+```
+
+```python
+def infer(resourceName: str, batchSz: int, requestType: str):
+    if requestType == "outlier":
+        rows = x_outlier[0:0+batchSz]
+    elif requestType == "drift":
+        rows = x_h1[0:0+batchSz]
+    else:
+        rows = x_ref[0:0+batchSz]
+    reqJson["inputs"][0]["data"] = rows.flatten().tolist()
+    reqJson["inputs"][0]["shape"] = [batchSz, rows.shape[1]]
+    headers = {"Content-Type": "application/json", "seldon-model":resourceName}
+    response_raw = requests.post(url, json=reqJson, headers=headers)
+    print(response_raw)
+    print(response_raw.json())
+```
+
+### Pipeline with model, drift detector and outlier detector
+
+```bash
+kubectl create -f ../../models/income.yaml -n ${NAMESPACE}
+```
+
+```
+model.mlops.seldon.io/income created
+
+```
+
+```bash
+kubectl wait --for condition=ready --timeout=300s model --all -n ${NAMESPACE}
+```
+
+```
+model.mlops.seldon.io/income condition met
+
+```
+
+```bash
+cat ../../pipelines/income-v1.yaml
+```
+
+```yaml
+apiVersion: mlops.seldon.io/v1alpha1
+kind: Pipeline
+metadata:
+  name: income-prod
+spec:
+  steps:
+    - name: income
+  output:
+    steps:
+    - income
+
+```
+
+```bash
+kubectl apply -f ../../pipelines/income-v1.yaml -n ${NAMESPACE}
+```
+
+```
+pipeline.mlops.seldon.io/income-prod configured
+
+```
+
+```bash
+kubectl wait --for condition=ready --timeout=300s pipeline --all -n ${NAMESPACE}
+```
+
+```
+pipeline.mlops.seldon.io/income-prod condition met
+
+```
+
+```python
+batchSz=1
+print(y_ref[0:batchSz])
+infer("income-prod.pipeline",batchSz,"normal")
+```
+
+```
+[0]
+<Response [200]>
+{'model_name': '', 'outputs': [{'data': [0], 'name': 'predict', 'shape': [1], 'datatype': 'INT64'}]}
+
+```
+
+```bash
+kubectl create -f ../../models/income-drift.yaml -n ${NAMESPACE}
+```
+
+```bash
+kubectl wait --for condition=ready --timeout=300s model --all -n ${NAMESPACE}
+```
+
+```
+model.mlops.seldon.io/income condition met
+model.mlops.seldon.io/income-drift condition met
+
+```
+
+```bash
+cat ../../pipelines/income-v2.yaml
+```
+
+```yaml
+apiVersion: mlops.seldon.io/v1alpha1
+kind: Pipeline
+metadata:
+  name: income-prod
+spec:
+  steps:
+    - name: income
+    - name: income-drift
+      batch:
+        size: 20
+  output:
+    steps:
+    - income
+
+```
+
+```bash
+kubectl apply -f ../../pipelines/income-v2.yaml -n ${NAMESPACE}
+```
+
+```yaml
+Warning: resource pipelines/income-prod is missing the kubectl.kubernetes.io/last-applied-configuration annotation which is required by kubectl apply. kubectl apply should only be used on resources created declaratively by either kubectl create --save-config or kubectl apply. The missing annotation will be patched automatically.
```

Comment:
:information_source: Sadly _not_ actually YAML, but the heuristic is to find something that looks like a word, followed by a colon, followed by anything that doesn't include colons - this is an unfortunate (but rare) false positive